### PR TITLE
Cape Town  Class 1 - Rustumb Marz - JavaScript Core 3 - Week 1

### PR DESCRIPTION
**Pull request by russmahn**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Comment by JarrodBen** [comment]
I think your boolean for the "read" checkbox bug is still there, there are two ways to fix it, quite an easy bug to miss.

File: mandatory/debugging-practice/script.js [hunk 1]
```diff
@@ -31,14 +31,18 @@ function submit() {
   if (
     title.value == null ||
     title.value == "" ||
+    author.value == null ||
+    author.value == "" ||
     pages.value == null ||
-    pages.value == ""
+    pages.value == "" ||
+    check.checked == null ||
+    check.checked == ""
```
Comment (JarrodBen) [inline mandatory/debugging-practice/script.js:39]: I think this will still throw out an alert "Please fill all in" even if the checkbox isn't checked.

**Review by JarrodBen — COMMENTED** [review]
Just check this for me in the browser as well (the below comment), all in all everything else seems great, nice debugging.

**Comment by douglas86** [comment]
There seems to be a bit of a bug, then browser is asking you to also check the read checkbox, that checkbox should be optional when entering complete the form. Other than that all seems to be working correctly. For this one 4 out of 10. There doesn't seem to be any other problems.

**Comment by douglas86** [comment]
Sorry, 5 out of 10.

**Pull request closed without merging**